.github/workflows/playwright.yml

```diff
@@ -0,0 +1,69 @@
+name: Playwright Tests
```

Comment:
I've not been able to test this file. 
Does anyone know how I can test this?

Reply:
You can use identical trigger as in the on-pr YAML file. Once you have verified that this file works, then we can revert the commit that added the trigger on PR. 😊